front/src/components/boxs/device-in-room/device-features/sensor-value/SensorDeviceFeature.jsx

```diff
@@ -43,7 +44,9 @@ const SensorDeviceType = ({ children, ...props }) => {
           )}`}
         />
       </td>
-      <td>{props.deviceFeature.name}</td>
+      <td>
+        <Text id={props.deviceFeature.name}>{props.deviceFeature.name}</Text>
```

Comment:
In my case I put same translation key  than device type by default so all feature name have a translation.
By default the name is translated and if you put a specific name (in edit page) I display this name.

do you prefer I not translate default name ? if yes I put name in english or other?

Reply:
I feel a bit skeptical with the fact to translate a name.
If you store the key in name attribute, so it's not really a name.
@Pierre-Gilles any opinion?

Reply:
The name will be displayed in any case at other places in the frontend (charts, dashboard, ..), so better display the same name here.

But I agree with the work @euguuu did here, it's a good idea to translate names

Could we use those translations at the device creation to put the name in the correct language at device creation ?

Reply:
In my case I save the key of translation in feature name (like feature type) so it's display with language of user as long as user change name of feature

Reply:
Sorry to come back to this, but since then the project has evolved on feature display name (cf https://github.com/GladysAssistant/Gladys/pull/1604)

I don't this Withings PR has to change the way it's currently done on the core side for all integrations